variables.tf

```diff
@@ -93,6 +90,11 @@ variable "vpc_tag_key_override" {
 # These parameters have reasonable defaults.
 # ---------------------------------------------------------------------------------------------------------------------
 
+variable "filename" {
+  description = "The path to the function's deployment package within the local filesystem."
+  default     = ""
```

Comment:
possible to leave this null and check that it's passed at all instead of misusing a zero val?

Reply:
will this look like `filename = var.filename != null ? var.filename : ""` ?
I'm getting an error when trying to run the examples:
```
 Error: Unsupported argument
│ 
│   on ../../main.tf line 4, in module "lambda":
│    4:   filename = var.filename != null ? var.filename : ""
│ 
│ An argument named "filename" is not expected here.
```

Reply:
just make sure the zero value is documented that an empty string is basically psuedo-null to fulfill the interface